draft-ietf-dnsop-svcb-https.md

```diff
@@ -151,10 +151,8 @@ Additional goals specific to HTTPS RRs and the HTTPS use-case include:
   alternative service endpoints
 * Obtain the {{!ECH}} keys associated with an alternative service endpoint
 * Support non-default TCP and UDP ports
-* Address a set of long-standing issues due to HTTP(S) clients not
-  implementing support for SRV records, as well as due to a limitation
-  that a DNS name can not have both CNAME and NS RRs
-  (as is the case for zone apex names)
+* Enable SRV-like benefits for HTTP(S), where SRV {{?SRV=RFC2782}} has not
+  been widely adopted
```

Comment:
I actually think this context (can't have a CNAME and NS RR at the zone apex) is helpful as this is one of the driving factors.  SRV is just one of the mechanisms that could do this.   Perhaps still include "..., such as allowing service delegation for a zone apex name where CNAMEs aren't allowed, ..." ?

Reply:
OK.  Rather than duplicate the statement about CNAME-at-apex, I added a pointer back to the CNAME-at-apex bullet point.